### PR TITLE
Add 'Starting a design with accessibility' post

### PR DESCRIPTION
**Pull request by SteveBarnett**

### CODE REVIEW

**Comment by SteveBarnett** [comment]
This will fix #1334. 🙃

File: src/posts/2021-08-25-starting-a-design-with-accessibility.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+tags:
+  - quicktip
+---
+
+Even better than [reviewing a design for accessibility](https://www.a11yproject.com/posts/2021-08-14-reviewing-a-design-for-accessibility/) is designing with accessibility in mind from the start. It helps us consider the wide range of people who use our product.
+
+Here are three things we can do to start a design with accessibility.
+
+## Do a text-only design
+
```
[the remark this answers is not in the document]
Comment (ericwbailey) [reply src/posts/2021-08-25-starting-a-design-with-accessibility.md:16]: Lovely! Thank you!

File: src/posts/2021-08-25-starting-a-design-with-accessibility.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+Writing the text of images means we’ll already have `alt` text (or something that’s a good start on it). It helps us be clear on what information (if any) the image is conveying.
+
+Writing the text for links highlights why “Click here” isn’t helpful link text. Link text should describe where the person goes when they follow the link.
+
+Writing the text for buttons highlights why "Submit” usually isn’t helpful button text. Button text should describe what happens when the person presses the button.
+
+Writing the names of form elements forces us to think of good labels. It forces us to think about the clearest name for each element.
+
+## Do an HTML-only design
+
```
[the remark this answers is not in the document]
Comment (ericwbailey) [reply src/posts/2021-08-25-starting-a-design-with-accessibility.md:52]: Nice!

**Review by ericwbailey — APPROVED** [review]
Love it. Thanks for being patient with the review, was changing jobs so I was a little distracted. 

Is the info we used last time to pay you still good to use?

**Comment by SteveBarnett** [comment]
Yay! Thanks so much for the help getting it into shape 💪 .

Re pay info: yes please! :)

**Comment by SteveBarnett** [comment]
Yay! Thanks so much for the help getting it into shape 💪 .

Re pay info: yes please! :)

**Comment by ericwbailey** [comment]
Sent! Thanks for another great post 🙂

**Pull request merged**